test.js

```diff
@@ -1581,6 +1581,75 @@ test('errors', function (t) {
     })
 })
 
+test('token in cookie', function (t) {
```

Comment:
I would add two more tests:
- jwt cookie without `fastify-cookie` => we should throw a clear message because right now we will crash with `Cannot read property 'cookiename' of undefined`
- jwt cookie + header => right now the header will win: is this the behaviour we want? I'm ok with so, but we should write a test for this use case